rtfm/source/_pages/usage/initializers.rst

```diff
@@ -0,0 +1,45 @@
+.. _Initializers:
+
+Initializers
+============
+
+  ``config/initializers/*.php`` files
+
+Remember the freaky ``functions.php`` file, the one where you would drop every
+bit of code external to the theme views (custom post types, taxonomies,
+wordpress filters, hooks, you name it?) That was just terrible, right?
+Well, forget it.
+
+Wordless lets you split your code into many modular initializer files, each
+one with a specific target:
+::
+
+  config/initializers
+  ├──── backend.php
+  ├──── custom_gutenberg_acf_blocks.php
+  ├──── custom_post_types.php
+  ├──── default_hooks.php
+  ├──── hooks.php
+  ├──── login_template.php
```

Comment:
Perhaps useless comment. the description below is missing

Reply:
Going to push an update